reference/fleet/roll-back-integration.md

```diff
@@ -0,0 +1,73 @@
+---
+navigation_title: Roll back an integration
+description: Roll back an Elastic Agent integration to the previously installed version, restoring the integration policies and configurations of the previous version.
+applies_to:
+  stack: ga 9.3
+  serverless: ga
+products:
+  - id: fleet
+  - id: elastic-agent
+---
+
+# Roll back an {{agent}} integration
+
+::::{note}
+This feature is available only for certain subscription levels. For more information, refer to [Elastic subscriptions]({{subscriptions}}).
+::::
+
+If you encounter issues after upgrading an integration, you can roll back the integration to the version installed before the upgrade. During the rollback action, the integration package and all associated integration policies and their configurations are automatically restored to the previously installed version.
+
```

Comment:
If you decide to emphasize "From Kibana" and "From Integrations page," a mini in-page TOC with links might be helpful.  Or even a sentence with links line.

Something like:

"You can roll back an integration from `<rollback-from-kibana-link>` or `<rollback-from-integration-page-link>`."